base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -2450,7 +2455,7 @@ impl BlockchainBackend for LMDBDatabase {
         let constants = self.consensus_manager.consensus_constants(height);
 
         // Get the current epoch for the height
-        let end_epoch = constants.block_height_to_epoch(height);
+        let end_epoch = self.consensus_manager.block_height_to_epoch(height);
         // Subtract the registration validaty period to get the start epoch
         let start_epoch = end_epoch.saturating_sub(constants.validator_node_validity_period_epochs());
```

Comment:
You have the constants above, why are you changing the one, but leaving the other one as be?